projects/opendr_ws/src/perception/scripts/heart_anomaly_detection.py

```diff
@@ -25,11 +25,12 @@
 
 class HeartAnomalyNode:
 
-    def __init__(self, input_topic="/ecg/ecg", prediction_topic="/opendr/heartanomaly", device="cuda", model='anbof'):
+    def __init__(self, input_ecg_topic="/ecg/ecg", output_heart_anomaly_topic="/opendr/heart_anomaly",
+                 device="cuda", model="anbof"):
         """
         Creates a ROS Node for heart anomaly (atrial fibrillation) detection from ecg data
-        :param input_topic: Topic from which we are reading the input array data
-        :type input_topic: str
+        :param input_ecg_topic: Topic from which we are reading the input array data
+        :type input_ecg_topic: str
         :param prediction_topic: Topic to which we are publishing the predicted class
         :type prediction_topic: str
```

Comment:
Mismatched docstring.

```suggestion
        :param output_heart_anomaly_topic: Topic to which we are publishing the predicted class
        :type output_heart_anomaly_topic: str
```